README.md

```diff
@@ -61,6 +111,65 @@ $zoneInfo = $client->getZoneInfo($username);
 print_r($zoneInfo);
 ```
 
+## Resource Impersonation
+Make sure you read the API documentation to enable and set the permissions.
```

Comment:
Maybe here it would be good to include what Resource Impersonation does (I assume it allows me to make a chance as someone else?). The idea behind that would be if someone is browsing this readme and hasnt seen the docs, they can get a brief summary and know whether or not this is something they need to pay attention to, or something that they don't need.

Reply:
Good call. I added a summary to the section.